Shaders/water_pass/water_pass.json

```diff
@@ -61,11 +61,6 @@
 					"name": "P",
 					"link": "_projectionMatrix",
 					"ifdef": ["_SSR"]
-				},
-				{
-					"name": "V3",
-					"link": "_viewMatrix3",
-					"ifdef": ["_SSR"]
```

Comment:
`V3` is still used in `water_pass.frag.glsl`, I don't think this should be removed.

Reply:
OK